tools/travis/run_tests.sh

```diff
@@ -34,7 +34,10 @@ if [ "$test" = "static" ]; then
 elif [ "$test" = "unit" ]; then
     dependencies="apt install -y git bzr subversion mercurial libnacl-dev libsodium-dev libffi-dev libapt-pkg-dev libarchive-dev python3-pip squashfs-tools xdelta3 && python3 -m pip install -r requirements-devel.txt -r requirements.txt codecov && apt install -y python3-coverage"
 elif [ "$test" = "integration" ] || [ "$test" = "plugins" ] || [ "$test" = "store" ] || [ "$test" = "containers" ] || [ "$test" = "snapd" ]; then
-    dependencies="apt install -y bzr curl git libnacl-dev libsodium-dev libffi-dev libapt-pkg-dev libarchive-dev mercurial python3-pip subversion squashfs-tools sudo snapd xdelta3 && python3 -m pip install -r requirements-devel.txt -r requirements.txt"
+    # snap install core exits with this error message:
+    # - Setup snap "core" (2462) security profiles (cannot reload udev rules: exit status 2
+    # but the installation succeeds, so we just ingore it.
+    dependencies="apt install -y bzr curl git libnacl-dev libsodium-dev libffi-dev libapt-pkg-dev libarchive-dev mercurial python3-pip subversion squashfs-tools sudo snapd xdelta3 && python3 -m pip install -r requirements-devel.txt -r requirements.txt && (snap install core || echo 'ignored error') && sudo snap install snaps-cache/snapcraft-pr$TRAVIS_PULL_REQUEST.snap --dangerous --classic"
```

Comment:
Might be a good idea to break this down a bit since it becomes hard to read. Though it doesn't have to be done in this branch.

Reply:
It always puzzles me how to break lines in bash. Is it only to put a \ at the end of the line inside the quotes? And then continue on the first column? agh, I hate bash.

Reply:
I hear yeah. Feel free to leave it as-is for now.